maquetado/setup/index.html

```diff
@@ -10,5 +10,12 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div class="tech">
+    <img src="assets/react_icon.svg" alt="ReactJS">
+    <h2>ReactJS</h2>
+    <p id="release">Release:2013</p>
```

Comment:
I think `id` is not needed here.

As this is kind of a title, what do you think of using something like `h4`?